src/lib/AnnouncementHeader.svelte

```diff
@@ -26,29 +26,27 @@
 
   .container {
     display: flex;
+    flex-direction: column;
     align-items: center;
     justify-content: start;
     width: auto;
-    background: #eeeeee;
```

Comment:
Not sure if that is an improvement, but it's barely noticable whether it's this or the other color.